feat: UIG-2689 - cypress component testing

ksvlaanderen opened the pull request:
Cypress component testing toegevoegd
Cypress tests van vl-side-navigation omgezet naar component tests

[Jira](https://www.milieuinfo.be/jira/browse/UIG-2689)
[Bamboo](https://www.milieuinfo.be/bamboo/browse/UIGOV-CUWC120)

Goldflow commented on libs/elements/src/side-navigation/stories/vl-side-navigation.stories-doc.mdx:
```diff
@@ -29,33 +28,26 @@ import { VlSideNavigation } from '@domg-wc/elements';
 ## Responsive variant
 
-Op mobiel zal de `vl-side-navigation` een knop bovenaan tonen die de gebruiker de mogelijkheid geeft het navigatie-menu te zien.
-Als de gebruiker scrollt, zal deze knop verdwijnen en komt na een korte delay een sticky knop met dezelfde functionaliteit. Deze knop blijft zichtbaar zolang de gebruiker scrollt in een gebied gerelateerd tot de `vl-side-navigation-reference`.
```
Waarom verwijderen van deze info?

ksvlaanderen replied:
Heb het teruggedraaid.

Goldflow reviewed (commented):
is het mogelijk om de commando('s) om cypress component tests te runnen ergens toe te voegen;
edit: zoals besproken - dit zijn native commando's dus moet niet per sé

Goldflow reviewed (dismissed):
excellent!

ksvlaanderen committed:
feat: UIG-2689 - cypress component testing

Cypress component testing toegevoegd
Cypress tests van vl-side-navigation omgezet naar component tests

mathiasbosman commented on cypress/fixtures/example.json:
Deze file nog nodig?

ksvlaanderen replied:
Deze file dient momenteel als voorbeeld en als gitkeep-file zodat de folder mee in de repo gecommit wordt.

mathiasbosman commented on libs/elements/src/side-navigation/vl-side-navigation.element.cy.ts:
```diff
@@ -0,0 +1,439 @@
+    });
+
+    it('should mount', () => {
+        cy.get('nav[is="vl-side-navigation"]');
```
Beetje jammer dat we hier niet op de custom tag kunnen zoeken; (stel dat men onderliggend de html zou wijzigen gaat dit breken)

maar is intrensiek door lit een probleem =( dus ik snap het

Goldflow replied:
ja op een bepaald moment converteren we alles naar Lit, dan kunnen we idd query doen op tag zelf ipv `is="..."`

ksvlaanderen replied:
Dat komt inderdaad doordat het een element is en geen component, ooit wordt dit omgevormd zodat we enkel nog maar web components hebben.

mathiasbosman commented on reporter-config.json:
Voor Bamboo neem ik aan?

ksvlaanderen replied:
Jes voor de build

mathiasbosman reviewed (commented):
Looking good if you ask me

kspeltix commented:
:tada: This PR is included in version 1.18.0 :tada:

The release is available on [GitHub release](https://github.com/milieuinfo/uigov-web-components/releases/tag/v1.18.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
